packages/web-pkg/src/components/AppBar/CreateSpace.vue

```diff
@@ -2,14 +2,14 @@
   <oc-button
     id="new-space-menu-btn"
     key="new-space-menu-btn-enabled"
-    v-oc-tooltip="$gettext('Create a new space')"
+    v-oc-tooltip="showTooltip ? $gettext('New space') : undefined"
```

Comment:
When you show a label:
- aria-label must not be present because the button label already says it all
- tooltip also can be removed in that case

When you're not showing a label:
- aria-label and tooltip are supposed to be the same content (And I prefer the longer `Create a new space` tooltip over the shorter one actually).


No other combination allowed a11y-wise :-)

Reply:
`Create a new space` wording would clash with the other create buttons (e.G New User) in the admin settings, adjusted the other remarks.